browser/BUILD.gn

```diff
@@ -7,6 +7,13 @@ import("//extensions/buildflags/buildflags.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 
 source_set("browser_process") {
+  visibility = [
```

Comment:
this stuff didn't technically need to be in here, but it helps protect from future issues